contributing/guide-observability.md

```diff
@@ -120,6 +120,56 @@ logs don't show up in production normally.
 For the question of what constitutes an error, errors should be actionable by a
 human. See the [Dave Cheney article] on this topic for some more discussion.
 
+## Good Errors
+
+An error message is good if it communicates
+
+1. what went wrong
+2. where it went wrong
+3. for which inputs
```

Comment:
note: obfuscation should potentially be considered if sensitive data is involved. This is a very broad topic though admittedly.

Reply:
see below (8)

Reply:
indeed